doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -692,7 +692,7 @@ Sensor samples
 Trusted Firmware-M (TF-M) samples
 ---------------------------------
 
-* Added support for the :ref:`zephyr:nrf9151dk_nrf9151` board for all TF-M samples.
+* Added support for the :ref:`zephyr:nrf9151dk_nrf9151` and the :ref:`nRF9161 DK <ug_nrf9161>` boards for all TF-M samples.
```

Comment:
```suggestion
* Added support for the :ref:`zephyr:nrf9151dk_nrf9151` and the :ref:`nRF9161 DK <ug_nrf9161>` boards for all TF-M samples (except for the :ref:`provisioning_image_net_core` sample).
```

Reply:
Also, add a bullet point after line 533 (under the Cryptography samples section)

Support for the :ref:`nRF9161 DK <ug_nrf9161>` board for the :ref:`crypto_test` sample.